sros2/sros2/api/__init__.py

```diff
@@ -143,8 +145,7 @@ def _write_cert(cert, cert_path, *, encoding=serialization.Encoding.PEM):
 
 
 def create_ca_conf_file(path):
-    with open(path, 'w') as f:
-        f.write("""\
+    conf_string = """\
```

Comment:
`textwrap.dedent` might be easier on the eyes, here.

Reply:
Yeah that occurred to me and I figured I'd keep the diff minimal to ease review

As this is already touching the conf file string, there's not much value in not making it look nicer right away so I'll add it to this PR :+1:

Reply:
Ah, beautiful, thank you!